pkgs/by-name/cu/cups-idprt/package.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenvNoCC
+, lib
+, fetchurl
+, cups
+, unzip
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "cups-idprt";
+  version = "1.4.2";
+
+  src = fetchurl {
+    name = "idprt_tspl_printer_linux_driver_v${version}.zip";
+    url = "https://www.idprt.com/prt_v2/files/down_file/id/131/fid/431.html"; # NOTE: This is NOT an HTML page, but a ZIP file
+    sha256 = "sha256-xY5E1RUNQwK0BO+0JMT7oHCAvSe79t7UU65IOzRQs/Q=";
+  };
+
+  buildInputs = [
+    cups
+  ];
+  nativeBuildInputs = [
+    unzip
+  ];
+
+  patchPhase = ''
+    runHook prePatch
+    sed -i 's/cupsModelNumber: 37155/cupsModelNumber: 20/g' ppd/*
+    sed -i 's/cupsFilter: "application\/vnd.cups-raster 100 raster-tspl"/cupsFilter: "application\/vnd.cups-raster 0 rastertolabel"/g' ppd/*
+    runHook postPatch
+  '';
+
+  dontConfigure = true;
+  dontBuild = true;
```

Comment:
Are those causing problems? if not, it is probably a good idea to just keep them enabled

Reply:
No, but I thought that best practice was to disable any build steps that aren't used.